modules/@apostrophecms/piece-type/index.js

```diff
@@ -938,6 +953,28 @@ module.exports = {
             throw self.apos.error('invalid', 'Document has neither slug nor title, giving up');
           }
         }
+      },
+
+      async getDraftToShare(req) {
```

Comment:
On all other routes we repeat the checks with `nofound`.
What is the difference here with `:_id/publish`/`:_id/unpublish` routes?

Once implemented, this will be part of the public API.

Reply:
`notfound` check is done in the `getDraftToShare` method.